uninstall.php

```diff
@@ -11,4 +11,32 @@
 	exit;
 }
 
-delete_option( 'perflab_modules_settings' );
+// For a multisite, delete the option for all sites (however limited to 100 sites to avoid memory limit or timeout problems in large scale networks).
+if ( is_multisite() ) {
+
+	$site_ids = get_sites(
+		array(
+			'fields'                 => 'ids',
+			'number'                 => 100,
+			'update_site_cache'      => false,
+			'update_site_meta_cache' => false,
+		)
+	);
+
+	foreach ( $site_ids as $site_id ) {
+		switch_to_blog( $site_id );
+		perflab_delete_plugin_option();
+	}
+	restore_current_blog();
```

Comment:
One remaining issue here which I had missed yesterday: `restore_current_blog()` must be called in every run of the `foreach` loop. This may sound confusing, and it certainly is. `restore_current_blog()` is a terrible name for the function, because it actually does not restore the current blog, but rather the previous blog, whether that was the current blog or another switched one.

You can see this also in a couple places in WordPress core: When looping through sites and switching to them, you always need to switch back ("restore") at the end of each loop cycle.
```suggestion
	foreach ( $site_ids as $site_id ) {
		switch_to_blog( $site_id );
		perflab_delete_plugin_option();
		restore_current_blog();
	}
```